README.md

```diff
@@ -86,6 +86,49 @@ const SomeComponent = () => {
 };
 ```
 
+## Additional features
+
+### Caching
+
+There is the option to invalidate the cache, meaning next time the hook is called it will fetch the data again.
+
+By setting a timeTillCacheInvalidate time in ms, as follows:
+
+**In `SomeHighLevelComponent.tsx`**
+
+```tsx
+import React from 'react';
+import { Provider } from 'react-redux';
+import { store } from './redux/store';
+import { State, Thing } from './types';
+
+const SomeComponent = () => {
+  <Provider store={store}>
+    <ReactUseFetchWithReduxProvider timeTillCacheInvalidate={1800000}>
```

Comment:
I'm still not great with useContext yet, but wouldn't the thing that we want to be using here be 

```
<ReactUseFetchWithRedux.Provider>
...
</ReactUseFetchWithRedux.Provider>
```

Looking at the code examples (and just how other providers are used in general, with the exception of react-redux's provider which is just called `Provider`) on https://reactjs.org/docs/hooks-reference.html#usecontext that seems to be the standard. I could be wrong here, though.

Reply:
I have done that in the ReactUseFetchWithReduxProvider file.

I think its better this way, you wont get confused when it doesn't work when the .provider is missed, 
The redux provider will be working the same way.

Reply:
I think, im going to check that

Reply:
Yes so in react-redux they have done:

```
function Provider({ store, context, children }) {
    blah blah blah ....
  return <Context.Provider value={contextValue}>{children}</Context.Provider>
}
```